src/__tests__/resources/asset.integration.spec.ts

```diff
@@ -0,0 +1,219 @@
+// Copyright 2019 Cognite AS
```

Comment:
That's because the whole file is new? The `/asset**s**.integration.spec.ts` was deleted

Reply:
I will restore `/assets.integration.spec.ts` as an empty file (so when merging it will just use v1 branch). 
This is testing `/classes/asset.ts` and not `/assets/assetsApi.ts`.

Reply:
Aha I see. Then you shouldn't need to think about `/assets.integration.spec.ts` at all